libs/platform/src/lib/form/form-generator/interfaces/dynamic-form-item.ts

```diff
@@ -237,6 +237,9 @@ export interface DynamicFormFieldItem {
 
     /** Horizontally align text inside the second column (Applicable for two columns layout) */
     secondaryTextAlignment?: TextAlignment;
+
+    /** Custom template used to build control body. */
+    controlTemplate: TemplateRef<any>;
```

Comment:
- [ ] TemplateRef is not imported;
- [ ] Property should be optional

Reply:
I do not like this. This interface is intended for use in every control, if you enhance select to use this, then other provided controls should use this as well.
For achieving custom stuff, I would go for [this approach](https://sap.github.io/fundamental-ngx/#/platform/form-generator#custom)

Reply:
I do not understand your proposal, I'm afraid. Select is not a custom control act. https://sap.github.io/fundamental-ngx/#/platform/wizard-generator > Wizard Generator Elements: select.
In addition, already 3 properties from select were added: "secondaryKey, showSecondaryText, secondaryTextAlignment" recently by mikerodonnell89 and he asked me to add this one myself as we figured this was missing in the first place.

Reply:
@littleamigo I created issue, which describes this. https://github.com/SAP/fundamental-ngx/issues/9349

It would be nice if we made this happen this way